src/main/resources/assets/jeresources/lang/ru_ru.json

```
@@ -1,129 +1,164 @@
-"jer.config.itemsPerColumn.title": "Кол-во предметов в столбце",
-"jer.config.itemsPerColumn.description": "Изменяет кол-во предметов в столбце JEI-просмотров подземелий и мобов",
-"jer.config.itemsPerRow.title": "Кол-во предметов в ряду",
-"jer.config.itemsPerRow.description": "Изменяет кол-во предметов в ряду JEI-просмотра подземелий",
-"jer.config.enchantsBlacklist.title": "Чёрный список зачарований",
-"jer.config.enchantsBlacklist.description": "Убирает зачарования из JEI-просмотра зачарований",
-"jer.config.diyData.title": "Использовать данные из модов",
-"jer.config.diyData.description": "Встроенный код совместимости не будет загружаться, и json-ы будут загружены (требуется перезапуск)",
-"jer.config.dimensionsBlacklist.title": "Черный список измерений",
-"jer.config.dimensionsBlacklist.description": "Убирает измерения из профилирующего сканирования JEI",
+{
+    "jer.config.itemsPerColumn.title": "Кол-во предметов в столбце",
+    "jer.config.itemsPerColumn.description": "Изменяет кол-во предметов в столбце JEI-просмотров подземелий и мобов",
+    "jer.config.itemsPerRow.title": "Кол-во предметов в ряду",
+    "jer.config.itemsPerRow.description": "Изменяет кол-во предметов в ряду JEI-просмотра подземелий",
+    "jer.config.enchantsBlacklist.title": "Чёрный список зачарований",
+    "jer.config.enchantsBlacklist.description": "Убирает зачарования из JEI-просмотра зачарований",
+    "jer.config.diyData.title": "Использовать данные из модов",
+    "jer.config.diyData.description": "Встроенный код совместимости не будет загружаться, и json-ы будут загружены (требуется перезапуск)",
+    "jer.config.dimensionsBlacklist.title": "Черный список измерений",
+    "jer.config.dimensionsBlacklist.description": "Убирает измерения из профилирующего сканирования JEI",
+	"jer.config.showDevData.title": "Показать дополнительную информацию",
+	"jer.config.showDevData.description": "В основном для создателей сборок. Функции включают в себя, но не ограничиваются средн. блоков на чанк, ID существ, ...",
+	"jer.config.hiddenTabs.title": "Скрыть вкладки JER",
+	"_comment": "# do not translate strings between quotes as they are used in code",
+	"jer.config.hiddenTabs.description": "Вкладки добавленные JER 'dungeon', 'enchantment', 'mob', 'villager', worldgen' и 'plant'",
```

Seems that a quote was missed.
```suggestion
	"jer.config.hiddenTabs.description": "Вкладки добавленные JER 'dungeon', 'enchantment', 'mob', 'villager', 'worldgen' и 'plant'",
```